### PR TITLE
Fix some linter warns

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: core/test/unit/src/config/template-contexts/base.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ describe("ConfigContext", () => {
     context("allowPartial=true", () => {
       it("should throw on missing key when allowPartial=true", async () => {
         const c = new TestContext({})
-        expectError(() => resolveKey(c, ["basic"], { allowPartial: true }), {
+        void expectError(() => resolveKey(c, ["basic"], { allowPartial: true }), {
```
Comment (Walther) [inline core/test/unit/src/config/template-contexts/base.ts:62]: Can we use `void` here or should we use `await` on these? I'm not 100% sure how this works with mocha, are the test assertions actually completed for the test cases?

Comment (vvagaytsev) [reply core/test/unit/src/config/template-contexts/base.ts:62]: Ah, you're right! The function can return a `Promise`, and we use `await` in the other code places.

Comment (vvagaytsev) [reply core/test/unit/src/config/template-contexts/base.ts]: Fixed in https://github.com/garden-io/garden/pull/4514/commits/1ca387cd3417278f78d069246da79dcbb87b2d4f

**Review by Walther — APPROVED** [review]
Looks good, thanks for the cleanup! ✨ 🧹

**Pull request merged**